lib/workload/stateless/filemanager/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3.1'
+
+services:
+  postgres:
+    build: database
+    container_name: filemanager_db
+    restart: always
+    environment:
+      - POSTGRES_DATABASE=filemanager
+      - POSTGRES_USER=filemanager
+      - POSTGRES_PASSWORD=filemanager
+      - PGPORT=4321
+    ports:
+      - "4321:4321"
```

Comment:
We tend to use just `compose.yml` these days. But, that's all good.! Just saying..

Reply:
Didn't know that! Seems to be change from 3 years ago https://github.com/docker/compose/releases/tag/1.28.6/.